canonical/vanilla-framework
# Fix favicon not rendering properly in google search.

## Done

Updates site to use 48x48 version of the Vanilla logo as a favicon, instead of the old 32x32 version. We are also now serving a favicon locally for search engines, following the practice done by [c.com](https://github.com/canonical/canonical.com/blob/f866a0da17f41e9e02f8a6f6786483ea040f5ce9/templates/base_index.html#L100-L104) & [u.com](https://github.com/canonical/ubuntu.com/blob/f3f3592e5a8d5dc6c1a33f7dc9cbbae2a4059f22/templates/templates/base.html#L52-L53).

See [Google favicon rendering documentation](https://developers.google.com/search/docs/appearance/favicon-in-search) for more information.

Fixes [WD-12464](https://warthogs.atlassian.net/browse/WD-12464), [MM thread](https://chat.canonical.com/canonical/pl/ja9ya3tgkjdpfkhgidt1x8xama)

## QA

- Open [demo](https://vanilla-framework-5183.demos.haus/)
- Verify favicon renders as expected.
- Open [locally served favicon](https://vanilla-framework-5183.demos.haus/static/images/favicons/vanilla_favicon_48px.png)
- Verify it also renders as expected.

## Google Recrawl
After this is merged, it may take a few days for the icon to change on Google search results. If it still hasn't been fixed after some time, we should [submit a recrawl request](https://developers.google.com/search/docs/crawling-indexing/ask-google-to-recrawl) which should fix the issue.

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical/vanilla-framework/wiki/Release-process#pre-release-tasks)


[WD-12464]: https://warthogs.atlassian.net/browse/WD-12464?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
